pkg/lib/cockpit.js

```diff
@@ -4415,6 +4415,12 @@ function factory() {
         return false;
     };
 
+    cockpit.assert = (predicate, message) => {
+        if (!predicate) {
+            throw new Error(`Assertion failed: ${message}`);
```

Comment:
I think this should get a `// not-covered: reason` annotation to satisfy coverage?

Reply:
ah, you didn't trigger /devel, so it can't complain

Reply:
I actually intend to cover this line when I get the tests working...